release.py

```diff
@@ -191,13 +195,10 @@ def parse_release_notes(release_notes: str) -> tuple[dict[str, list[tuple[str, s
         if match := re.match(r'^\* (\w+): (.*) by @\w+ in (.*)', line.strip()):
             category = match.group(1)
             if category in categories:
-                description = match.group(2).strip().capitalize()
+                description = match.group(2).strip()
+                description = description[0].upper() + description[1:]
                 pr_link = match.group(3).strip()
                 categories[category].append((description, pr_link))
-            else:
```

Comment:
Now that there's no warning, I think we should add `perf` into the known categories, and probably `refactor` as well.

For `revert`, in some ways it would be nice to try to find the reverted commit and skip both of them, but that's more complicated and would only work if the revert was in the same release. So I think we should include `revert` as well.

That means that the only commits that we skip mentioning are `chore` (I would personally include `ci` as well, but we've traditionally included that, so probably still should).

Reply:
I don't have a strong opinion on this, I'm OK both ways, with or without `perf`/`refactor`/`revert`. But I'm not going to make this change in this PR, mainly because according to our discussion in Frankfurt, we agreed to use only 5 categories -- ['Categorise by “Features”, “Fixes”, “Documentation”, “Tests”, “CI” (in useful order)'](https://docs.google.com/document/d/1fkoqp1A2SMiOnwHubOJVRkuD1GpvcYxSDmfExYIZO88/edit?tab=t.0).